src/app/frontend/uicomponents/helpsection/helpsection.scss

```diff
@@ -0,0 +1,21 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.kd-help-section > * {
+  width: 50% !important;
```

Comment:
I really wanted to avoid using CSS for enforcing a layout of the *kd-help-section*, but I don't see a direct alternative, because the transcluded content is seen "as a whole", i.e. I can't directly pre-define the flex properties of the contained children (can I?). @bryk any better ideas? 

As an alternative I could specify flex on the spot, but that would defeat the purpose of the whole kd-help-section wrapper to some extent. I really want to enforce a preset layout here.